examples/java/src/main/java/glide/examples/ClusterExample.java

```diff
@@ -0,0 +1,165 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.examples;
+
+import static glide.api.models.configuration.RequestRoutingConfiguration.SimpleMultiNodeRoute.ALL_NODES;
+
+import glide.api.GlideClusterClient;
+import glide.api.logging.Logger;
+import glide.api.models.ClusterValue;
+import glide.api.models.commands.InfoOptions;
+import glide.api.models.configuration.GlideClusterClientConfiguration;
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.exceptions.ClosingException;
+import glide.api.models.exceptions.ConnectionException;
+import glide.api.models.exceptions.ExecAbortException;
+import glide.api.models.exceptions.TimeoutException;
+
+import java.util.concurrent.CancellationException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+public class ClusterExample {
+
+    /**
+     * Creates and returns a <code>GlideClusterClient</code> instance. This function initializes a
+     * <code>GlideClusterClient</code> with the provided list of nodes.
+     *
+     * @return A <code>GlideClusterClient</code> connected to the discovered nodes.
+     * @throws CancellationException if the operation is cancelled.
+     * @throws ExecutionException if the client fails due to execution errors.
+     * @throws InterruptedException if the operation is interrupted.
+     */
+    public static GlideClusterClient createClient() throws CancellationException, ExecutionException, InterruptedException {
+        String host = "localhost";
+        Integer port = 6379;
+        // GLIDE is able to detect all cluster nodes and connect to them automatically
```

Comment:
remove this comment and add it to the function description, again - please use the python description